mantidproject/mantid
# Remove ornl reflectometry interfaces

Removing the unused ORNL reflectometry interfaces from Mantid to reduce maintenance and testing.

**To test:**
This should be tested at ORNL in particular, but also any other facility that has an interest in the RefM and REFL workings within Mantid.

1. Check that the REFL, REFM and REF SF interfaces are gone from the interfaces->reflectometry menu, and the underlying code has gone
1. The code in Scripts/Interface/reduction_gui/reduction/reflectometer directory in place as it is needed by LRAutoReduction.py
1. Check the other Interfaces based on the reduction_gui framework still work (Almost all of the ORNL interfaces).

Fixes #21141

**Release Notes** 
 docs/source/release/... /lowq.rst

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
